bin/ream/Cargo.toml

```
@@ -21,6 +21,7 @@ ream-executor.workspace = true
 ream-network-spec.workspace = true 
 ream-p2p.workspace = true 
 ream-storage.workspace = true
+ream-rpc={workspace = true}
```

```suggestion
ream-rpc.workspace = true
ream-storage.workspace = true
```

Alphabetical order and the brackets are redundent